hedy_content.py

```diff
@@ -49,67 +49,26 @@
     'hi': 'HI'
 }
 
-class LevelDefaults:
-  def __init__(self, language):
-    self.language = language
-    self.keyword_lang = "en"
-    self.keywords = YamlFile.for_file(f'content/keywords/{self.keyword_lang}.yaml').to_dict()
-    self.levels = YamlFile.for_file(f'content/level-defaults/{self.language}.yaml')
-
-  def set_keyword_language(self, language):
-      if language != self.keyword_lang:
-          self.keyword_lang = language
-          self.keywords = YamlFile.for_file(f'content/keywords/{self.keyword_lang}.yaml')
-
-  def max_level(self):
-    all_levels = sorted(self.levels.keys()) # We should sort this to make sure the max_level returned is correct
-    max_consecutive_level = 1
-    previous_level = 0
-    for level in all_levels:
-      if level == previous_level + 1:
-        previous_level = level
-        max_consecutive_level = level
-      else:
-        return previous_level
-    return max_consecutive_level
-
-  def get_defaults_for_level(self, level):
-    #grabs level defaults from yaml and converts to DefaultValues type
-    default_values = copy.deepcopy(self.levels[level])
-
-    # Sometimes we have multiple text and example_code -> iterate these and add as well!
-    extra_examples = []
-    for i in range(2, 10):
-        extra_example = {}
-        if default_values.get('intro_text_' + str(i)):
-            extra_example['intro_text'] = default_values.get('intro_text_' + str(i)).format(**self.keywords)
-            default_values.pop('intro_text_' + str(i))
-            if default_values.get('example_code_' + str(i)):
-                extra_example['example_code'] = default_values.get('example_code_' + str(i)).format(**self.keywords)
-                default_values.pop('example_code_' + str(i))
-            extra_examples.append(extra_example)
-        else:
-            break
-    default_values['extra_examples'] = extra_examples
-    for k,v in default_values.items():
-        if isinstance(v, str):
-            default_values[k] = v.format(**self.keywords)
-    default_type = {
-      "level": str(level),
-    }
-    default_type.update(**default_values)
-    return DefaultValues(**default_type)
-
-  def get_defaults(self, level):
-    """Return the level defaults for a given level number."""
-
-    return copy.deepcopy(self.levels.get(int(level), {}))
-
-
-class NoSuchDefaults:
-  def get_defaults(self):
-    return {}
-
+ADVENTURE_ORDER = [
+    'default',
```

Comment:
here it is called default, but in the PR description I also saw "intro" can we keep that to one term? I like defaults because in my mind these things will always be level defaults, but for other people, I think `intro` is clearer.

Reply:
This is only the dictionary key in the YAMLs, so end users won't see it. We can change it to 'intro', but do have to change the keys in all YAMLs as well.

Reply:
If possible I would prefer that (can also be done in a separate PR)